Modules/Projects.js

```diff
@@ -0,0 +1,357 @@
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    linkTolive: 'link 4',
+    linkTosource: 'link 4',
+    button: true,
+  },
+  {
+    id: 'p6',
+    name: 'Professional Art Printing Data',
+    description: `A daily selection of privately personalized reads; 
+   no accounts or sign-ups required. Has been the industry's standard.`,
+    featuredImage: null,
+    backgroundImage: 'Works-Images/sec-2-bg.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    linkTolive: 'link 5',
+    linkTosource: 'link 5',
+    button: true,
+  },
+  {
+    id: 'p7',
+    name: 'Professional Art Printing Data',
+    description: `A daily selection of privately personalized reads; 
+    no accounts or sign-ups required. Has been the industry's standard.`,
+    featuredImage: null,
+    backgroundImage: 'Works-Images/sec-2-bg.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    linkTolive: 'Works-Images/sec-2-bg.png',
+    linkTosource: 'link 5',
+    button: true,
+  },
+];
+
+function loadModals(Object) {
+  const Modal = document.querySelector('#modal');
+  const modalTitle = document.querySelector('#modal-title');
+  const titleText = modalTitle.textContent;
+  const ModalList = document.querySelector('#modal-icons');
+  const listHTML = ModalList.innerHTML;
+  const ModalPoster = document.querySelector('#modal-poster');
+  const posterSRC = ModalPoster.src;
+  const ModalDescription = document.querySelector('#Modal-Description');
+  const descripText = ModalDescription.textContent;
+  const CloseModal = document.querySelector('#modal-close');
+  const BlurBackground = document.querySelector('#overlay');
+
+  if (Object.name !== null) {
+    modalTitle.innerText = Object.name;
+  }
+  if (Object.featuredImage !== null) {
+    ModalPoster.src = Object.featuredImage;
+  }
+  if (Object.technologies !== null && Array.isArray(Object.technologies)) {
+    ModalList.innerHTML = '';
+    Object.technologies.forEach((tech) => {
+      const techLi = document.createElement('li');
+      techLi.innerText = tech;
+      ModalList.appendChild(techLi);
+    });
+  }
+  if (Object.description !== null) {
+    ModalDescription.innerText = Object.description;
+  }
+  CloseModal.addEventListener('click', () => {
+    Modal.style.visibility = 'hidden';
+    BlurBackground.style.visibility = 'hidden';
+    modalTitle.innerText = titleText;
+    ModalPoster.src = posterSRC;
+    ModalList.innerHTML = listHTML;
+    ModalDescription.innerText = descripText;
+  });
+  Modal.style.visibility = 'visible';
+  BlurBackground.style.visibility = 'visible';
+}
+
+function LoadProjects() {
+  const MainContainer = document.querySelector('#Portfolio');
+  let Projects;
+  let ModalTitle;
+  if (window.innerWidth < DesktopWidth) {
+    Projects = MobileProjects;
+  } else {
+    Projects = DeskProjects;
+    ModalTitle = document.querySelector('#modal-title');
+    ModalTitle.textContent = 'Keeping track of hundreds  of components website';
+  }
```

Comment:
[Optional] I noticed that you have different project arrays for mobile and desktop. Please note that as this is one webpage, all your projects are supposed to be the same regardless of which device is used to view it. Your current implementation is responsive, however, it is also resource-heavy and makes your webpage slow as the browser constantly checks device width and rendering projects. I highly suggest you refactor your webpage to use a single project array and use media queries to define how the modal looks on mobile and desktop